esmvaltool/_data_finder.py

```diff
@@ -59,7 +60,7 @@ def select_files(filenames, start_year, end_year):
     return selection
 
 
-def replace_tags(path, variable):
+def replace_tags(path, variable, j):
```

Comment:
Please provide a default value of `None` for `j` and remove the dummy value of `0` from the rest of the code.